core/src/main/java/com/graphhopper/util/GHUtility.java

```diff
@@ -723,6 +727,20 @@ public static long calcMillisWithTurnMillis(Weighting weighting, EdgeIteratorSta
         }
     }
 
+    public static void runConcurrently(List<Callable<String>> callables, int threads) {
```

Comment:
```suggestion
    public static void runConcurrently(List<Callable<String>> callables, int threads, String threadPrefix) {
```

Would be nice if we could remove the Thread renaming from the Callables and let the ThreadFactory of the Executor handle this.